tests/utils.py

```diff
@@ -32,8 +32,8 @@ def fake_tx(
     outputs: Optional[TxOutput] = None,
     tokens: Optional[bytes] = None,
 ) -> Transaction:
-    tx_inps = inputs or [fake_input() for _ in range(fake.pyint(1, 10))]
-    tx_outps = outputs or [fake_output() for _ in range(fake.pyint(1, 10))]
+    tx_inps = inputs or [fake_input() for _ in range(fake.pyint(1, 5))]
```

Comment:
Why are you reducing the number of inputs and outputs?

Reply:
When debugging some issues that were solved in another PR I needed to decrease the number of inputs and outputs so I could manually debug some issues.